pkg/crd/operator/v1alpha1/configuration.go

```diff
@@ -49,6 +49,8 @@ type KubermaticConfigurationSpec struct {
 	Auth KubermaticAuthConfiguration `json:"auth"`
 	// FeatureGates are used to optionally enable certain features.
 	FeatureGates sets.String `json:"featureGates,omitempty"`
+	// MachineController configures the Machine Controller
+	MachineController MachineControllerConfiguration `json:"machineController,omitempty"`
```

Comment:
I'm not sure if this is the right place for this struct. Since the machine-controller is a user cluster component, it might make sense to put this field in the `KubermaticUserClusterConfiguration` struct.

Reply:
I thought about that to move it to seed configuration (since seed manager deploys MC), let me take a look again, maybe your suggestion is better.